common/constants.go

```diff
@@ -326,6 +326,7 @@ const (
 	ContentTypeText = "text/plain"
 	ContentTypeXML  = "application/xml"
 	ContentTypeForm = "application/x-www-form-urlencoded"
+	ContentTypeCSV  = "text/csv"
```

Comment:
move to edgecentral.go to avoid the future conflict

Reply:
moved to xpertconstants.go